peerberrypy/api.py

```diff
@@ -318,14 +318,14 @@ def get_investments(
             self,
             quantity: int,
             start_page: int = 0,
-            max_date_of_purchase: int = None,
-            min_date_of_purchase: int = None,
+            max_date_of_purchase: Optional[int] = None,
+            min_date_of_purchase: Optional[int] = None,
             max_interest_rate: Optional[decimal.Decimal] = None,
             min_interest_rate: Optional[decimal.Decimal] = None,
             max_invested_amount: Optional[decimal.Decimal] = None,
             min_invested_amount: Optional[decimal.Decimal] = None,
-            countries: list = None,
-            loan_types: list = None,
+            countries: Optional[list[str]] = None,
+            loan_types: Optional[list[str]] = None,
```

Comment:
Instead of having `list[str]`, it's better to have `List[str]` to ensure backward compatibility with older Python versions. You can use the List from the typing module.